uw-research-computing/transfer-data-researchdrive.md

```diff
@@ -11,9 +11,7 @@ would like to know more.
 
 ## A. Pre-Requisites
 
-In order to follow the steps in this guide, you need: 
-1. Access to a ResearchDrive share, either as PI or member of your PI's group. 
-2. Access to CHTC's "transfer" server, `transfer.chtc.wisc.edu`. 
+In order to follow the steps in this guide, you need access to a ResearchDrive share, either as PI or member of your PI's group. 
```

Comment:
I also submitted a PR with changes for this guide. 😂 
https://github.com/CHTC/chtc-website-source/pull/353

I think the easiest will be for me to resubmit my changes as a pull request against your changes and you can review. I'll try to do that today (12/21).